### DISCUSSION TITLE
Difficulty with sending a string type from React to Unity

### DISCUSSION THREAD

**Discussion by alexhaloETH** [jeffreylanters/react-unity-webgl#442] [Help Wanted!] [answered]
Hello, this might be a basic question but I am having a bit of a hard time sending a String back to my unity project from React.
I am able to send an Int and I am also aware that a string from JS or TS is different from the one in Unity but i don't understand where and how to do the conversions, to be honest.

Unity code
`public void SetAddress(string address) 
    {

        //address.ToString();

        addressText.text = addressText.text + ": 0x" +  address;
    }`

TSx code   (address is the string i am trying to send)
`
    sendMessage("UIscript", "SetAddress", address)`

In the TS code bit, I also have a function that sends an int back to unity so I know everything works.

Thanks in advance

**Comment by alexhaloETH** [comment, marked as answer]
Ok never it looks like the variable address itself is the issue, feel free to delete this discussion post

sorry for the bother, have a nice day

**Reply by jeffreylanters** [reply]
No worries, glad you've solved it. Best of luck with your project! If you've any other questions, feel free to open a new discussion.
